src/garage/tf/algos/npo.py

```diff
@@ -155,10 +161,19 @@ def init_opt(self):
                                       self._policy_opt_inputs),
                                   constraint_name='mean_kl')
 
-        return dict()
-
     def optimize_policy(self, itr, samples_data):
-        """Optimize policy."""
+        """Optimize policy.
+
+        Args:
+            itr (int): Iteration number.
+            samples_data (dict): Processed sample data.
+                See process_samples() for details.
+
+        Raises:
```

Comment:
i don't see the raise statement anywhere?

Reply:
My fault. Fixed.